_posts/2022-10-13-access-the-k8s-api-from-functions.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Access the Kubernetes API from a function
+description: "We show you how to access the Kubernetes API from an OpenFaaS function"
```

Comment:
We'll show you have to access any part of the Kubernetes API from a function securely, with a granular ServiceAccount.